From 4eff6f931037d58c49e58dc5fde584834d662593 Mon Sep 17 00:00:00 2001
From: Leonardo Graboski Veiga <leonardo.veiga@toradex.com>
Date: Wed, 6 Nov 2019 13:26:50 -0300
Subject: [PATCH 2/2] pasta-demo: use older devicetree name

That is compliant with the Embedded Linux BSP 3.0b2 release of the Linux
kernel by Toradex.
---
 conf/machine/apalis-imx8.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/machine/apalis-imx8.conf b/conf/machine/apalis-imx8.conf
index 72ce51e..6ab3660 100644
--- a/conf/machine/apalis-imx8.conf
+++ b/conf/machine/apalis-imx8.conf
@@ -25,7 +25,7 @@ LOADADDR = ""
 # auto-serial-console there
 USE_VT = "0"
 
-KERNEL_DEVICETREE = "freescale/fsl-imx8qm-apalis-eval.dtb"
+KERNEL_DEVICETREE = "freescale/fsl-imx8qm-apalis.dtb"
 
 UBOOT_SUFFIX = "bin"
 UBOOT_CONFIG ??= "sd"
-- 
2.17.1

